test/powershell/Language/Scripting/DeserializedTypeConversion.Tests.ps1

```diff
@@ -0,0 +1,284 @@
+﻿Describe "Tests conversion of deserialized types to original type using object properties." -Tags "CI" {
+
+    BeforeAll {    
```

Comment:
fixed

---
In reply to: [77376739](https://github.com/Powershell/powershell/pull/2118#discussion_r77376739) [](ancestors = 77376739)